src/components/panel/panel.js

```diff
@@ -126,6 +126,9 @@ angular
  *     outside the panel to close it. Defaults to false.
  *   - `escapeToClose` - `{boolean=}`: Whether the user can press escape to
  *     close the panel. Defaults to false.
+ *   - `postClose` - `{function=}`: Function that is called after close
```

Comment:
{function(panelRef, string)=} is the correct type, especially since this has params.